### PR TITLE
Add footer to all pages

### PR DESCRIPTION
**Pull request by shandyk18**
A simple footer was added to the bottom of all the pages. 

Screenshot 2020-06-24 at 12 05 45 PM - Display 2

Small stylistic changes were made: change text font (Google font), container-fluid to fill up page, move text from bottom to top on main page

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,44 +4,45 @@
...
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
   </head>
   <body>
-    <div id="content" class="container">
-      <div class="container p-3 my-3 bg-dark text-white">
+    <div id="content" class="container-fluid -fluid ">
+      <div class="container-fluid -fluid  p-3 my-3 bg-dark text-white">
```
Comment (wujingya) [inline portfolio/src/main/webapp/index.html:16]: extra -fluid?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,44 +4,45 @@
...
                 <a class="nav-link" href="https://www.github.com/shandyk18" target="_blank">GitHub</a>
-                </li>
+              </li>
             </ul>
-            </nav>
+          </nav>
         </div>
         <div class="col-8">
-          <div id="overview" class="container">
+          <div id="overview" class-fluid ">
```
Comment (wujingya) [inline portfolio/src/main/webapp/index.html:45]: did you mean class="container-fluid"?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -62,9 +59,23 @@ <h3>Overview</h3>
...
           </div>
         </div>
       </div>
+      <footer class="page-footer font-xsmall">
+        <div class="container-fluid -fluid text-center text-md-left p-3 my-3 bg-dark text-white">
+          <div class="row">
+            <hr class="clearfix w-100 d-md-none pb-3">
+            <div class="col-md-3 mb-md-0 mb-3">
+              <h5 class="text-uppercase">Contact</h5>
+              <p>shandyk@google.com</p>
```
Comment (wujingya) [inline portfolio/src/main/webapp/index.html:74]: optional: can we make this a <a> tag with "mailto:shandyk@google.com"?

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shandyk18) [reply portfolio/src/main/webapp/index.html:16]: I used a replace all and didn't notice some errors. Fixed!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (shandyk18) [reply portfolio/src/main/webapp/index.html:45]: Fixed!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (shandyk18) [reply portfolio/src/main/webapp/index.html:74]: Done!

**Pull request merged**